src/sequence_batch.cc

```diff
@@ -30,12 +30,12 @@ uint32_t SequenceBatch::LoadBatch() {
     if (length > 0) {
       kseq_t *sequence = sequence_batch_[sequence_index];
       std::swap(sequence_kseq_->seq, sequence->seq);
-      ReplaceByEffectiveRange(sequence->seq);
+      ReplaceByEffectiveRange(sequence->seq, true);
```

Comment:
Can you change this to `ReplaceByEffectiveRange(sequence->seq, /*is_seq=*/true);`? This helps a lot when we look at the code in the future, and avoid potential bugs. Also for other this function calls. I have identified so many code that has this pattern and the parameters were passed in the wrong way. Please also do this for future code.